tests/http2-bugfixes/test.yaml

```diff
@@ -14,3 +14,17 @@ checks:
       count: 0
       match:
         event_type: anomaly
+# check gzip decompresser
+  - filter:
+      feature: HTTP2_DECOMPRESSION
```

Comment:
Can you add an example of this to the main readme?

Reply:
@inashivb this is not exactly the same format as in https://redmine.openinfosecfoundation.org/issues/4185
Does it matter ?

(will update the README after this answer)

Reply:
@catenacyber no this is OK. feature is a good key name here.

Reply:
Added the example in README in next PR #398